templates/demo-store/app/routes/($locale).account.tsx

```diff
@@ -221,67 +229,71 @@ const CUSTOMER_QUERY = `#graphql
     $language: LanguageCode
   ) @inContext(country: $country, language: $language) {
     customer(customerAccessToken: $customerAccessToken) {
+      ...CustomerDetails
+    }
+  }
+
+  fragment CustomerDetails on Customer {
```

Comment:
Extracting this as a fragment so that we can import the `CustomerDetailsFragment` generated type.